scripts/percona-release.sh

```diff
@@ -314,8 +317,10 @@ function enable_alias {
 }
 #
 function check_setup_command {
-  if [[ -n ${2} ]]; then
-    echo "* \"setup\" command does not accept additional options!"
+  if [[ "$1" == "-y" || "${!#}" == "-y" ]]; then
+      export INTERACTIVE=no
+  elif [[ -n ${2} ]]; then
+    echo "* \"setup\" command supports only \"-y\" option as the automatic answer \"yes\" for all questions."
```

Comment:
let's leave `echo "* \"setup\" command supports only \"-y\"` only. 
the next call is show help where user will get infor regarding -y option

Reply:
Done